clients/python/play.py

```diff
@@ -1,5 +1,38 @@
+import json
+import sys
+
+from game.api import API
+from game.models import Game
+
+
 def main():
-    print("TODO")
+    team_secret = fetch_team_secret()
```

Comment:
I had a PR where I changed the word `team` to `player` everywhere in the project. Can you provide me the place where there is still `team` please? I'll accept your PR